### PR TITLE
upgrade to xunit 2.2.0

### PR DESCRIPTION
**Pull request by shhsu**

### CODE REVIEW

File: src/Benchmarks.Framework/project.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     },
     "Microsoft.AspNetCore.Testing": "1.1.0-*",
     "Microsoft.NETCore.Platforms": "1.0.1-*",
-    "xunit": "2.1.0",
-    "xunit.runner.reporters": "2.1.0"
+    "xunit": "2.2.0-*",
+    "xunit.runner.reporters": "2.2.0-*"
```
Comment (pranavkm) [inline src/Benchmarks.Framework/project.json]: Do you actually need this?

File: test/TestApp.Test/project.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
         "System.Threading": "4.0.11-*",
 		
         "Microsoft.NETCore.Platforms": "1.0.1-*",
-        "xunit": "2.1.0",
+        "xunit": "2.2.0-*",
         "dotnet-test-xunit": "1.0.0-*"
```
Comment (pranavkm) [inline test/TestApp.Test/project.json]: Update to `dotnet-test-xunit: 2.2.0-*". Similarly elsewhere.

**Comment by pranavkm** [comment]
:shipit:

File: src/Benchmarks.Framework/project.json (hunk 1 above)
Comment (shhsu) [inline src/Benchmarks.Framework/project.json]: Yup, build actually failed without it

File: test/TestApp.Test/project.json (hunk 2 above)
Comment (shhsu) [inline test/TestApp.Test/project.json]: Ok, I will try it out and then do a test run

File: src/Benchmarks.Framework/project.json (hunk 1 above)
Comment (shhsu) [inline src/Benchmarks.Framework/project.json]: Doesn't hurt to match the versions right?

Comment (pranavkm) [inline src/Benchmarks.Framework/project.json]: No, that's fine.

**Comment by DavidObando** [comment]
:shipit:

**Comment by DavidObando** [comment]
The build is passing after these changes, go ahead and merge @shhsu. Thanks!

**Pull request merged**